runtime/doc/message.txt

```diff
@@ -402,12 +402,6 @@ existing abbreviation.  All variations of this command give the same message:
 ":cunabbrev", ":iunabbrev", etc.  Check for trailing white space.
 
 >
-  /dev/dsp: No such file or directory
-
-Only given for GTK GUI with Gnome support.  Gnome tries to use the audio
-device and it isn't present.  You can ignore this error.
-
-							*E31*  >
```

Comment:
`E31` is the tag for the text below it, so shouldn't be removed. It doesn't look like the `/dev/dsp` error message ever had a tag... or at least it got deleted a while ago.